docs/developers/guard-accounts.md

```diff
@@ -0,0 +1,134 @@
+
+If the sent ```SetGuardian``` transaction is an unguarded one, the user will be able to see ```pendingGuardian``` set ([along with the activation epoch](/sdk-and-tools/rest-api/addresses#get-address-guardian-data)). On the other hand, if the transaction is a guarded one, the new guardian will become directly active.
+
+## Guard account
+
+A guard account transaction can be issued by the account and processed by the protocol only if the account has an ```activeGuardian``` (see [guardian-data fields](/sdk-and-tools/rest-api/addresses#get-address-guardian-data)) already set (which implies that the **20 epochs** since sending the ```SetGuardian``` already passed). When the account is guarded, any transaction issued by the owner of the account would be ignored unless it also carries the signature of its set guardian. This transaction could be sent either directly from the user or also through a relayer, if the user does not have the egld required for the transaction fee. 
+
+:::note
+The exception is the ```setGuardian``` transaction which can either:
+
+- be executed immediately if it is co-signed by the active guardian and **cleans up any pending guardian activation**;
+
+- not be confirmed by the guardian but in this case the guardian can only be set with an activation time in future (e.g the same 20 epochs from the initial setting of a guardian).
+:::
+
+:::note
+The ```GuardAccount``` transaction should clear any pending guardian.
+:::
+
+## Sending guarded (co-signed) transactions
+
+Sending a guarded transaction will follow the same process of using the [Send Transaction](/sdk-and-tools/rest-api-/transactions#send-transaction) endpoint. In order for the transaction to be accepted by the protocol (when account is guarded and more than 20 epochs passed after sending the ```SetGuardian``` transaction), the optional fields ```guardian``` and ```guardianSignature``` have to be completed along with the proper version and option.
+
+:::info
+For a Guarded Transaction the **Version** must be set to **2** and **Options** must have the **second bit** set.
+:::
+
+A guarded transaction would look like:
+
+```json
+{
+    "nonce": 2,
+    "value": "0",
+    "receiver": "erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th",
+    "sender": "erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th",
+    "guardian": "erd1k2s324ww2g0yj38qn2ch2jwctdy8mnfxep94q9arncc6xecg3xaq6mjse8",
+    "gasPrice": 1000000000,
+    "gasLimit": 1177500,
+    "data": "U2V0R3VhcmRpYW5AYjEzYTAxNzQyM2MzNjZjYWZmOGNlY2ZiNzdhMTI2MTBhMTMwZjQ4ODgxMzQxMjJjNzkzN2ZlYWUwZDZkN2QxN0A3NTc1Njk2NA==", #base64 representation of Seting a Guardian
```

Comment:
Perhaps explain what this data means as base64 decoded string and what every argument is supposed to mean